subprojects/enterprise/src/integTest/groovy/org/gradle/internal/enterprise/GradleEnterprisePluginCheckInIntegrationTest.groovy

```diff
@@ -86,4 +89,33 @@ class GradleEnterprisePluginCheckInIntegrationTest extends AbstractIntegrationSp
         then:
         plugin.serviceCreatedOnce(output)
     }
+
+    @IgnoreIf({ GradleContextualExecuter.configCache })
```

Comment:
can you explain this ?
What does this executer do, and why do you need to remove the test from being executed with it ?
(forget my lack of knowledge here)

Reply:
I took it from here:

https://github.com/gradle/gradle/blob/b0bb465ce06f49c2840016371893d006c09189ee/subprojects/enterprise/src/integTest/groovy/org/gradle/internal/enterprise/GradleEnterprisePluginConfigurationCachingIntegrationTest.groovy#L27-L29